dotcom-rendering/src/client/adaptiveSite.ts

```diff
@@ -13,6 +13,8 @@ import { recordExperiences } from './ophan/ophan';
 export const shouldAdapt = async (): Promise<boolean> => {
 	if (isServer) return false;
 	if (window.location.hash === '#adapt') return true;
+	// used to disable adaptive for e2e testing
+	if (window.location.hash === '#noadapt') return false;
```

Comment:
The opposite of “adapt” could be:

- resist
- preserve
- sustain
- worsen
- bypass